v1/content.yaml

```diff
@@ -152,6 +152,13 @@ paths:
             To get a 200 response instead, supply `false` to the `redirect` parameter.
           type: boolean
           required: false
+        - name: Accept-Language
+          in: header
+          description: >
+            The desired language variant code for wikis where LanguageConverter is enabled.
+            Example: `sr-el` for latin trascription of the Serbian language.
```

Comment:
* Latin, * transcription

Reply:
Sorry, didn't quite understand what you meant here

Reply:
"Latin" should be upper case. "trascription" is a typo, missing an n. Should be "transcription".

Reply:
oh.. didn't notice the typo, so the `*` made me confused. Sorry.

Reply:
My fault, I should have been clearer, sorry. :-)